### ISSUE TITLE
Include Migration Guide

### ISSUE THREAD

**Issue by wynneplaga** [facebook/buck2#212]
There are several references in the docs to a migration guide file for migrations from buck. Ideally that document could be included in the repository.

**Comment by ndmitchell** [comment]
The document is only really useful for migrating from Buck1 inside the Meta internal repo. If you are trying to migrate from Buck1, I'll go through and sanitise it and see what might be helpful to make public. Are you currently on Buck1?

**Comment by wynneplaga** [comment]
Yes, curious to see if it's not an incredibly heavy lift to swap in if it might be worth recommending

**Comment by ndmitchell** [comment]
I've copied the most interesting bits into a gist so you can have it now: https://gist.github.com/ndmitchell/e0314ebf264707eccf36c37736e55e5c

It's a bit awkward having two versions of this, one with all the public stuff, one with the private (not very private, mostly irrelevant) stuff interspersed. Not sure how to go longer term...
